components/ModEditHealthcareProfessionalSection.vue

```diff
@@ -1,233 +1,199 @@
+                                text-primary-text text-sm font-normal font-sans placeholder-primary-text-muted"
+                    @change=" () => healthcareProfessionalsStore
+                        .updateHealthcareProfessionalNameValues(healthcareProfessionalsStore.selectedHealthcareProfessionalData,
+                                                                sectionFields.nameLocale)"
                 >
-                    <div
-                        class="flex justify-center items-center rounded-lg p-3  border-2 border-primary"
-                        :data-testid="`mod-submission-list-item-${index + 1}`"
+                    <option
+                        v-for="(locale, index) in Locale"
+                        :key="`${locale}-${index}`"
                     >
-                        <div>
-                            <div class="flex font-bold">
-                                <SVGProfileIcon
-                                    role="img"
-                                    alt="profile icon"
-                                    title="profile icon"
-                                    class="profile-icon w-6 h-6 stroke-primary stroke-1 inline mx-1"
-                                />
-                                <span>{{ healthcareProfessionalName.lastName }}</span>
-                                <span class="mx-2">{{ healthcareProfessionalName.firstName }}</span>
-                                <span v-show="healthcareProfessionalName.middleName">
-                                    {{ healthcareProfessionalName.middleName }}
-                                </span>
-                            </div>
-                            <div
-                                class="w-fit px-2 py-[1px] my-2 border border-primary/40 rounded-full
-                                    shadow text-sm text-center hover:bg-primary/20 transition-all"
-                            >
-                                {{ healthcareProfessionalName.locale }}
-                            </div>
-                        </div>
-                        <span
-                            class="flex items-center justify-center
-                                cursor-pointer font-bold text-secondary text-sm"
-                            @click="() => handleRemoveHealthcareProfessionalName(index)"
-                        >
-                            <SVGTrashCan class="flex items-center justify-center w-6 h-6" /> {{
-                                $t('modHealthcareProfessionalSection.delete') }}
-                        </span>
-                    </div>
+                        {{ locale }}
+                    </option>
+                </select>
+                <button
+                    v-show="isEditSubmissionScreen"
+                    type="button"
+                    class="bg-tertiary text-white font-bold py-2 px-4 my-2 rounded w-56"
```

Comment:
It looks like this class was carried over, but it's a good time to change hard coded colors like`text-white` to a theme color like `text-primary-text` or `text-primary-text-inverted`